classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
+  has_many :students
+
+  def initialize(label)
+    @label = label
+  end
+
+  def add_student(student)
+    student.classroom = self
+  end
```

Comment:
- [ ] To successfully establish the `has-many relationship`, all students should be added to a student array in the `Classroom` class. 

e.g. 

  ```
def initialize(label)
    @label = label
    @students = []
  end

```
  ```
def add_student(student)
    @student << student
    student.classroom = self
  end
```